networkx/algorithms/community/louvain.py

```diff
@@ -118,6 +118,7 @@ def louvain_communities(
     See Also
     --------
     louvain_partitions
+    :any:`leiden_communities`
```

Comment:
I'm not sure we should "See Also" to functions that aren't implemented by NX.
This could get confusing.

Reply:
I don't have a general recommendation other than we should be thoughtful when doing this.

The sentiment I have heard about Leiden over the last couple of years is that when people reach for Louvain, they would probably get better results if they used Leiden, so they should at least consider using Leiden. Although the "See Also" doesn't give such a strong suggestion, it does provide a clear hint, which is what "See Also" is for IMHO. I think it makes sense for Leiden to "See Also" to Louvain in case they don't have a backend installed that implements Leiden.